resources/assets/components/StatusButton/status-button.scss

```diff
@@ -7,7 +7,11 @@ $accepted-green: #80E85D;
 
 .button {
   &.-accepted {
-    background: $light-gray;
+    background: $white;
+    color: $light-gray;
+    border-color: $light-gray;
+    border-width: 3px;
```

Comment:
thanks, @lkpttn ! Removed the border when the button is active: 
![screen shot 2017-06-09 at 11 21 49 am](https://user-images.githubusercontent.com/9019452/26982286-ef54b840-4d05-11e7-817b-2f81d732d574.png)